site/src/pages/docs/server-docservice.mdx

```diff
@@ -20,7 +20,7 @@ sb.http(8080);
 sb.service("/hello", THttpService.of(new MyThriftHelloService()));
 
 // Add a gRPC service which implements 'GrpcHelloService'.
-// Unlike Thrift, you must enable gRPC-Web and unframed requests explicitly.
+// Unlike Thrift, you must enable unframed requests explicitly.
 sb.service(GrpcService.builder()
                       .addService(new MyGrpcHelloService())
                       .supportedSerializationFormats(GrpcSerializationFormats.values())
```

Comment:
I think we can remove this line as well. 😄

Reply:
`.enableUnframedRequests(true)` is not needed?

Reply:
GrpcSerializationFormats.values() is set by default. 😁

Reply:
Ah, you mentioned L26.